lib/looker-sdk/version.rb

```diff
@@ -26,6 +26,6 @@ module LookerSDK
 
   # Current version
   # @return [String]
-  VERSION = "0.0.7"
+  VERSION = "0.0.8"
```

Comment:
Not sure if we're supposed to bump the version manually, or if the rake release task does it for us

Reply:
it needs to be bumped manually but the rake release automates pushing the git tag to github and the gem to rubygems.